configserver/pom.xml

```diff
@@ -291,11 +291,6 @@
         <groupId>com.yahoo.vespa</groupId>
         <artifactId>bundle-plugin</artifactId>
         <extensions>true</extensions>
-        <configuration>
-          <!-- TODO: Vespa > 8: remove importPackage when the jackson-jaxrs-json-provider bundle is no longer installed in jdisc -->
-          <!--   added to ensure using the same class as orchestrator, core-dump-reporter, provision-controller and controller-clients -->
-          <importPackage>com.fasterxml.jackson.jaxrs.json;version="[2.12.6,3)"</importPackage>
-        </configuration>
```

Comment:
@gjoranv Is this related to dataformat-xml?

Reply:
No, this cannot be removed now.

Reply:
Reverted this change back.